### ISSUE TITLE
add timetable to schedule turn on / off 

### ISSUE THREAD

**Issue by M4rcDev** [Jugendhackt/LithiumBuoy#8]

**Comment by moritz157** [comment]
AlarmTab?

_Sent from my Samsung SM-G930F using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by M4rcDev** [comment]
Yes, but we need to make it indepent from the mobile app, so it can be done without the smartphone. Maybe improve the nodemcu software to schedule it there.
